Test error message is confusing

Failed tests like: https://travis-ci.com/sql-machine-learning/sqlflow/jobs/268910624 have error messages:

```
=== RUN   TestStdinParseOnly
time="2019-12-19T12:51:12Z" level=info msg="sqlflowCmd: run locally" package=sql
time="2019-12-19T12:51:15Z" level=error msg="runSQLProgram error: failed: exit status 1\n==========Program==========\n\nfrom sqlflow_submitter.tensorflow.train import train, TF_VERSION_2\nimport tensorflow as tf\nif TF_VERSION_2:\n    from tensorflow.keras.optimizers import *\nelse:\n    from tensorflow.train import *\ntry:\n    import sqlflow_models\nexcept:\n    pass\n\nfeature_column_names = [\n\"sepal_length\",\n\n\"sepal_width\",\n\n\"petal_length\",\n\n\"petal_width\",\n]\n\nfeature_metas = dict()\n\nfeature_metas[\"sepal_length\"] = {\n    \"feature_name\": \"sepal_length\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nfeature_metas[\"sepal_width\"] = {\n    \"feature_name\": \"sepal_width\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nfeature_metas[\"petal_length\"] = {\n    \"feature_name\": \"petal_length\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nfeature_metas[\"petal_width\"] = {\n    \"feature_name\": \"petal_width\",\n    \"dtype\": \"float32\",\n    \"delimiter\": \"\",\n    \"shape\": [1],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\n\nlabel_meta = {\n    \"feature_name\": \"class\",\n    \"dtype\": \"int64\",\n    \"delimiter\": \"\",\n    \"shape\": [],\n    \"is_sparse\": \"false\" == \"true\"\n}\n\nmodel_params=dict()\n\nmodel_params[\"hidden_units\"]=[10,20]\n\nmodel_params[\"n_classes\"]=3\n\n\nfeature_columns = {\"feature_columns\": [tf.feature_column.numeric_column(\"sepal_length\", shape=[1]),\ntf.feature_column.numeric_column(\"sepal_width\", shape=[1]),\ntf.feature_column.numeric_column(\"petal_length\", shape=[1]),\ntf.feature_column.numeric_column(\"petal_width\", shape=[1])]}\n\ntrain_max_steps = 0\ntrain_max_steps = None if train_max_steps == 0 else train_max_steps\n\ntrain(is_keras_model=\"false\" == \"true\",\n    datasource=\"mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0\",\n    estimator=tf.estimator.DNNClassifier,\n    select=\"\"\"SELECT *\nFROM iris.train\n\"\"\",\n    validate_select=\"\"\"SELECT *\nFROM iris.train\n\"\"\",\n    feature_columns=feature_columns,\n    feature_column_names=feature_column_names,\n    feature_metas=feature_metas,\n    label_meta=label_meta,\n    model_params=model_params,\n    metric_names=\"Accuracy\".split(\",\"),\n    save=\"model_save\",\n    batch_size=1,\n    epochs=2,\n    verbose=0,\n    train_max_steps=train_max_steps,\n    eval_start_delay_secs=0,\n    eval_throttle_secs=0,\n    save_checkpoints_steps=100,\n    log_every_n_iter=10,\n    is_pai=\"false\" == \"true\",\n    pai_table=\"\")\n\n==========Output==========\nvalidation select not empty\nTraceback (most recent call last):\n  File \"<stdin>\", line 106, in <module>\n  File \"/root/go/src/sqlflow.org/sqlflow/python/sqlflow_submitter/tensorflow/train.py\", line 259, in train\n    classifier = tf.estimator.add_metrics(classifier, metrics.get_tf_metrics(metric_names))\nUnboundLocalError: local variable 'metrics' referenced before assignment\n" package=sql
--- FAIL: TestStdinParseOnly (5.27s)
panic: interface conversion: interface {} is sql.EndOfExecution, not string [recovered]
	panic: interface conversion: interface {} is sql.EndOfExecution, not string
goroutine 34 [running]:
testing.tRunner.func1(0xc00038b500)
	/usr/local/go/src/testing/testing.go:874 +0x3a3
panic(0x1074320, 0xc0003cec60)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
sqlflow.org/sqlflow/cmd/repl.TestStdinParseOnly(0xc00038b500)
	/root/go/src/sqlflow.org/sqlflow/cmd/repl/repl_test.go:282 +0xdde
testing.tRunner(0xc00038b500, 0x1252fe0)
	/usr/local/go/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:960 +0x350
FAIL	sqlflow.org/sqlflow/cmd/repl	5.379s
```

The message `panic: interface conversion: interface {} is sql.EndOfExecution, not string` is confusing, may need a better way to deal with panics.

--- a/cmd/repl/repl_test.go
+++ b/cmd/repl/repl_test.go
@@ -292,8 +292,8 @@ INTO sqlflow_models.mymodel;`
 		case error:
 			var s []string
 			for e := lastResp.Front(); e != nil; e = e.Next() {
-				if _, ok := e.Value.(string); ok {
-					s = append(s, e.Value.(string))
+				if ss, ok := e.Value.(string); ok {
+					s = append(s, ss)
 				}
 			}
 			a.Fail(strings.Join(s, "\n"))

--- a/cmd/repl/repl_test.go
+++ b/cmd/repl/repl_test.go
@@ -292,7 +292,9 @@ INTO sqlflow_models.mymodel;`
 		case error:
 			var s []string
 			for e := lastResp.Front(); e != nil; e = e.Next() {
-				s = append(s, e.Value.(string))
+				if _, ok := e.Value.(string); ok {
+					s = append(s, e.Value.(string))
+				}
 			}
 			a.Fail(strings.Join(s, "\n"))
 		}

